argon/libs/flash_controller/src/SPI.cpp

```diff
@@ -71,7 +75,7 @@ namespace spi
         {
             std::uint32_t chunkSize = std::min(size, static_cast<size_t>(0x10000));
 
-            std::uint32_t transferred;
+            uint32 transferred;
```

Comment:
why?

Reply:
Because of typedef inside ftdi library messing with standard types